cmd/main.go

```diff
@@ -45,17 +39,26 @@ func run() error {
 	return nil
 }
 
-func init() {
+func initRunnerConfig(c *runner.RunConfig) {
+	var opts = bootstrap.Options{}
 	// for storage and service discovery
 	flags.AddConsulFlags(rootCmd, &opts)
+	// always use consul for storage and service discovery
+	c.Options = opts
```

Comment:
the problem in this is that opts is copied by value to c.Options. 
It's value gets populated in main by `rootCmd.Execute()` 
which means that the new values will never be copied to c.Options.

I think we can either make c.Options take a pointer to bootstrap.Options, or make this copy in a function that's called after main (run() and the other RunE callback)